src/Reflection/Annotations/AnnotationsMethodsClassReflectionExtension.php

```diff
@@ -66,6 +66,9 @@ private function findClassReflectionWithMethod(
 				$parameters,
 				$methodTags[$methodName]->isStatic(),
 				$this->detectMethodVariadic($parameters),
+				$classReflection->hasNativeMethod('__call')
+					? $classReflection->getNativeMethod('__call')->getThrowType()
+					: null,
```

Comment:
For static methods this should look at `__callStatic` instead

Reply:
Thanks, fixed